### PR TITLE
(partial) update users guides

### PR DESCRIPTION
**Pull request by ajm188**
This is a subset of the changes for `vtctlclient => vtctldclient` needed on the website, as outlined in #1182.

I still need to do the rest of the files underneath `configuration-advanced/`, but this is a large enough diff as-is that I wanted to split it out.

### CODE REVIEW

File: content/en/docs/15.0/user-guides/configuration-advanced/resharding.md [hunk 1]
```diff
@@ -175,8 +175,21 @@ for i in 400 401 402; do
...
+  done;
+  vtctldclient GetTablets --keyspace customer --shard $shard | wc -l | grep -q "3" || (echo "Timed out waiting for tablets to be up in customer/$shard" && exit 1)
+
+  # Wait for a primary tablet to be elected in the shard
+  for _ in $(seq 0 200); do
+    vtctldclient GetTablets --keyspace customer --shard $shard | grep -q "primary" && break
+    sleep 1
+  done;
+  vtctldclient GetTablets --keyspace customer --shard $shard | grep "primary" || (echo "Timed out waiting for primary to be elected in customer/$shard" && exit 1)
+done;
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/configuration-advanced/resharding.md]: This is now complex enough that I think we should just tell them to run the provided shell script.
Morgan's original idea was that people should learn the internal commands being run by the script but this is a bit too much.

File: content/en/docs/15.0/user-guides/configuration-advanced/resharding.md [hunk 2]
```diff
@@ -185,9 +198,9 @@ This process starts the reshard operation. It occurs online, and will not block
 
 ```bash
 # With Helm and Local Installation
-vtctlclient Reshard -- --source_shards '0' --target_shards '-80,80-' Create customer.cust2cust
+vtctldclient LegacyVtctlCommand -- Reshard --source_shards '0' --target_shards '-80,80-' Create customer.cust2cust
 # With Operator
-vtctlclient Reshard -- --source_shards '-' --target_shards '-80,80-' Create customer.cust2cust
+vtctldclient LegacyVtctlCommand -- Reshard --source_shards '-' --target_shards '-80,80-' Create customer.cust2cust
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/configuration-advanced/resharding.md]: "LegacyVtctlCommand" looks bad. We have to do this only because we have not yet implemented Reshard and MoveTables on the vtctld server API.
I'd rather keep vtctlclient for these commands.

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md [hunk 3]
```diff
@@ -14,8 +14,8 @@ If using a custom `init_db.sql` that omits `SET sql_log_bin = 0`, then InitShard
 The command for `InitShardPrimary` is as follows:
 
 ```text
-vtctlclient \
-  InitShardPrimary -- \
+vtctldclient \
+  InitShardPrimary \
   --force \
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md:19]: @GuptaManan100 is it time to mark this as deprecated?
Line 37 
```
`InitShardPrimary` will soon be deprecated. This action will be performed automatically by VTOrc once it is released as production-ready.
```
@ajm188 no changes needed in this PR. If we do this it can be a separate PR.

File: content/en/docs/15.0/user-guides/migration/cutover.md [hunk 4]
```diff
@@ -37,20 +37,20 @@ vtctlclient --server vtctld.host:15999 GetRoutingRules > /var/tmp/routingrules.b
 To begin MoveTables run the following command:
 
 ```sh
-vtctlclient --server vtctld.host:15999 MoveTables -- --source sourcekeyspace --tables 'table1,table2,table3' Create targetkeyspace.workflowname
+vtctldclient --server vtctld.host:15999 LegacyVtctlCommand -- MoveTables --source sourcekeyspace --tables 'table1,table2,table3' Create targetkeyspace.workflowname
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/migration/cutover.md]: My earlier comments about VRepl commands are relevant here too.

File: content/en/docs/15.0/user-guides/migration/cutover.md [hunk 5]
```diff
@@ -337,19 +337,19 @@ After you have tried the above command(s) with `--dry_run` remove just that flag
 - RDONLY:
   
 ```
-vtctlclient --server vtctld.host:15999 SwitchReads -- --tablet_types=rdonly targetkeyspace.workflowname
+vtctldclient --server vtctld.host:15999 LegacyVtctlCommand -- SwitchReads --tablet_types=rdonly targetkeyspace.workflowname
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/migration/cutover.md]: SwitchReads/SwitchWrites have been deprecated. It's supposed to be `SwitchTraffic`.
Maybe we need @mattlord or @rohit-nayak-ps to rewrite this page.

File: content/en/docs/15.0/user-guides/migration/materialize.md [hunk 6]
```diff
@@ -110,7 +110,7 @@ And now we can proceed to the `Materialize` step(s).
 We will run two `Materialize` operations, one for each copy/view of the `corder` table we will be creating.  We could combine these two operations into a single `Materialize` operation, but we will keep them separate for clarity.
 
 ```bash
-$ vtctlclient Materialize '{"workflow": "copy_corder_1", "source_keyspace": "commerce", "target_keyspace": "commerce", "table_settings": [{"target_table": "corder_view", "source_expression": "select * from corder"}]}'
+$ vtctldclient LegacyVtctlCommand -- Materialize '{"workflow": "copy_corder_1", "source_keyspace": "commerce", "target_keyspace": "commerce", "table_settings": [{"target_table": "corder_view", "source_expression": "select * from corder"}]}'
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/migration/materialize.md]: VRepl commands again.
Maybe we should just revert all changes to this file and cutover.md

File: content/en/docs/15.0/user-guides/migration/move-tables.md [hunk 7]
```diff
@@ -124,8 +124,24 @@ for i in 200 201 202; do
  CELL=zone1 KEYSPACE=customer TABLET_UID=$i ./scripts/vttablet-up.sh
 done
 
-vtctlclient InitShardPrimary -- --force customer/0 zone1-200
-vtctlclient ReloadSchemaKeyspace customer
+# Set the correct durability policy for the keyspace
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/migration/move-tables.md]: Let's replace the full script listing with how to run the script.

File: content/en/docs/16.0/user-guides/configuration-advanced/resharding.md [hunk 8]
```diff
@@ -175,8 +175,21 @@ for i in 400 401 402; do
  SHARD=80- CELL=zone1 KEYSPACE=customer TABLET_UID=$i ./scripts/vttablet-up.sh
 done
 
-vtctlclient InitShardPrimary -- --force customer/-80 zone1-300
-vtctlclient InitShardPrimary -- --force customer/80- zone1-400
+for shard in "-80" "80-"; do
```
Comment (deepthi) [inline content/en/docs/16.0/user-guides/configuration-advanced/resharding.md:178]: run script.

**Review by deepthi — COMMENTED** [review]
I left my feedback on the 15.0 version of the docs

File: content/en/docs/15.0/user-guides/configuration-advanced/resharding.md (hunk 1 above)
Comment (ajm188) [reply content/en/docs/15.0/user-guides/configuration-advanced/resharding.md]: oh, i just copied what was in the local examples, what is the provided script?

File: content/en/docs/15.0/user-guides/configuration-advanced/resharding.md (hunk 2 above)
Comment (ajm188) [reply content/en/docs/15.0/user-guides/configuration-advanced/resharding.md]: works for me

File: content/en/docs/15.0/user-guides/configuration-advanced/resharding.md (hunk 1 above)
Comment (deepthi) [reply content/en/docs/15.0/user-guides/configuration-advanced/resharding.md]: 302_new_shards.sh

File: content/en/docs/15.0/user-guides/migration/move-tables.md (hunk 7 above)
Comment (deepthi) [reply content/en/docs/15.0/user-guides/migration/move-tables.md]: 201_customer_tablets.sh

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md (hunk 3 above)
Comment (GuptaManan100) [reply content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md:19]: Yes, I think we can do it. With VTOrc GA and a required component, nothing should be required to initialize a shard.

**Review by deepthi — APPROVED** [review]
LGTM
Let's merge this so we can unblock follow up PRs.
